recipes/ncpol2sdpa/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ncpol2sdpa" %}
+{% set version = "1.12.2" %}
+{% set sha256 = "11233a8950e3e4918d19c90c41bcf259b17f2daa645b05432d5e1c51a3bdb0f7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:   https://pypi.python.org/packages/b7/0b/b0842e5b2f58743530f4a6de1fcdaef9884b73f17ff46106ca9ceed1d960/{{ name }}-{{ version }}.tar.gz
```

Comment:
For some reason, it does not work. I get 404. Package is always updated with `python setup.py sdist upload -r pypi`.